src/server/src/main/java/io/cassandrareaper/AppContext.java

```diff
@@ -46,6 +47,7 @@
   public SchedulingManager schedulingManager;
   public JmxConnectionFactory jmxConnectionFactory;
   public ReaperApplicationConfiguration config;
+  public Cryptograph cryptograph;
```

Comment:
nit-pick: can we do inversion-of-control (dependency-injection through the constructors) rather than service lookup via the context. I believe the instance created in ReaperApplication can be passed in to the needed classes via new constructor parameters.